dev/puppet-server.conf.sample

```diff
@@ -30,8 +30,11 @@ webserver: {
 # configure the mount points for the web apps
 web-router-service: {
     "puppetlabs.services.ca.certificate-authority-service/certificate-authority-service": "/puppet-ca"
-    "puppetlabs.services.master.master-service/master-service": "/puppet"
+    "puppetlabs.services.master.master-service/master-service": {master-routes:       "/puppet"
 
+                                                                 # The following controls 404 errors and should not
+                                                                 # be modified
+                                                                 invalid-in-puppet-4: "/" }
```

Comment:
@jeffmccune This should only catch requests that don't have the `/puppet` or `/puppet-ca` prefixes. Any other invalid request (say, `/puppet/v3/unknown`) will return the normal 404 error message.

I think that's a valuable thing to test, however, so I'll add tests to ensure that that is true.